linux/tools.Dockerfile

```diff
@@ -5,16 +5,16 @@
 
 # To build yourself locally, override this location with a local image tag. See README.md for more detail
 
-ARG IMAGE_LOCATION=cdpxlinux.azurecr.io/artifact/b787066e-c88f-4e20-ae65-e42a858c42ca/official/azure/cloudshell:1.0.20200727.1.base.master.40d4519a
+ARG IMAGE_LOCATION=cdpxlinux.azurecr.io/artifact/b787066e-c88f-4e20-ae65-e42a858c42ca/buddy/azure/cloudshell:1.0.20201009.1.base.add-cbld.5113b5b1
 
 # Copy from base build
 FROM ${IMAGE_LOCATION}
 
 # Install latest Azure CLI package. CLI team drops latest (pre-release) package here prior to public release
 # We don't support using this location elsewhere - it may be removed or updated without notice
-RUN wget -nv https://azurecliprod.blob.core.windows.net/cloudshell-release/azure-cli-latest.deb \
-  && dpkg -i azure-cli-latest.deb \
-  && rm -f azure-cli-latest.deb
+RUN wget -nv https://azurecliprod.blob.core.windows.net/cloudshell-release/azure-cli-latest-buster.deb \
```

Comment:
Curious: what's the buster version

Reply:
Azure-cli has packages for different linux distros. This is the one which works with Debian (closest match to CBL-D). Contains the same version as the other package.